src/components/storagePools/storagePoolList.jsx

```diff
@@ -45,7 +46,8 @@ export class StoragePoolList extends React.Component {
         const actions = (<CreateStoragePoolAction loggedUser={loggedUser} libvirtVersion={libvirtVersion} />);
 
         return (
-            <Page groupProps={{ sticky: 'top' }}
+            <WithDialogs>
+                <Page groupProps={{ sticky: 'top' }}
                   isBreadcrumbGrouped
                   breadcrumb={
```

Comment:
Just a nitpick, but this now looks a bit misintended. Strange that compiler doesn't throw an error "Expected intendation of A, but found B"

Reply:
Yeah... I have jus added the `WithDialogs` wrapper and then let eslint take care of the indentation... it doesn't always seem to do a great job... Fixed now.